ui/src/alerting/utils/checkValidate.ts

```diff
@@ -0,0 +1,16 @@
+import {Threshold} from 'src/types'
+import {isNaN} from 'lodash'
```

Comment:
There is a very subtle difference between lodash's isNaN and the global window.isNaN. https://lodash.com/docs/4.17.15#isNaN

lodash isNaN is strictly checking if the value is both a number and equal to NaN, whereas the global window.isNaN will check if the value is undefined.

Do we want to check for undefined here as well? If so, consider using the global window.isNaN

Reply:
thanks!